specifyweb/stored_queries/execution.py

```diff
@@ -384,7 +384,17 @@ def createPlacemark(kmlDoc, row, coord_cols, table, captions, host):
 
     return placemarkElement
 
-
+def test_qb(request):
+    with models.session_context() as session:
+        prep_col = func.max(getattr(models.Preparation, models.Preparation._id))
+        query = select([prep_col]
+                       ).select_from(models.Preparation).where(models.Preparation.CollectionObjectID == models.CollectionObject.collectionObjectId).correlate(models.CollectionObject).as_scalar()
+        new_query = session.query(models.CollectionObject._id).select_from(models.CollectionObject).filter(models.CollectionObject.CollectionID == request.specify_collection.id)
+        new_query = new_query.add_column(query)
+        new_query = new_query.group_by(query)
+        str_query = str(new_query.statement.compile(compile_kwargs={"literal_binds": True}))
+        rr = list(new_query)
+    raise Exception('aha')
```

Comment:
@acwhite211 @CarolineDenis I added this snippet of code for debugging. Replace `query` by `query.label(None)` in 
```
new_query = new_query.group_by(query)
```
and see that going to `localhost/api/test_error/` the column error is no longer thrown. 


TODO for you: Get rid of this function, and corresponding code in views.py